packages/iTwinUI-react/package.json

```diff
@@ -6,6 +6,14 @@
   "main": "cjs/index.js",
```

Comment:
If we are using `exports`, maybe the `main` is overridden and not needed?

Reply:
Well, I thought its needed for lower version of node and potentially for backwards compatibility if some bundlers fail with it
https://nodejs.org/docs/latest-v16.x/api/packages.html#package-entry-points

but not sure.. we could risk getting rid of it ? But again..safety reasons if `exports` is not supported..